script.js

```diff
@@ -0,0 +1,117 @@
+$(document).ready(function(){
```

Comment:
this works! not required because your script tag is before the closing body, but not wrong